source/MaterialXCore/Node.h

```diff
@@ -77,16 +77,19 @@ class Node : public InterfaceElement
     /// @{
 
     /// Return the first implementation for this node, optionally filtered by
-    /// the given target name.
+    /// the given target and language names.
     /// @param target An optional target name, which will be used to filter
     ///    the implementations that are considered.
+    /// @param language An optional language name, which will be used to filter
+    ///    the implementations that are considered.
     /// @return An implementation for this node, or an empty shared pointer if
     ///    none was found.  Note that a node implementation may be either an
     ///    Implementation element or a NodeGraph element.
-    InterfaceElementPtr getImplementation(const string& target = EMPTY_STRING) const
+    InterfaceElementPtr getImplementation(const string& target = EMPTY_STRING,
```

Comment:
Also extended node to allow taking in language to be consistent with the nodedef option.

Reply:
Good idea.